tests/test_black.py

```diff
@@ -264,6 +264,14 @@ def test_function2(self) -> None:
         black.assert_equivalent(source, actual)
         black.assert_stable(source, actual, black.FileMode())
 
+    @patch("black.dump_to_file", dump_to_stderr)
+    def test_function3(self) -> None:
```

Comment:
It's fine to have another test file, but could you name it something less generic, like `test_function_trailing_comma`?